components/remote_transmitter.rst

```diff
@@ -183,6 +183,12 @@ Configuration variables:
 
 This :ref:`action <config-action>` sends an NEC infrared remote code to a remote transmitter.
 
+.. note::
+
+    In version 2021.11, the order of transferring bits was corrected from MSB to LSB in accordance with the NEC standard.
+    Therefore, in earlier versions of the configuration, it is necessary to reverse the order of the address and command bits.
```

Comment:
Suggested wording:
Therefore, if the the configuration file has come from an earlier version of ESPhome, it is necessary to reverse the order of the address and command bits when moving to 2021.11 or above.